frameworks/kafka/tests/test_kerberos_authz.py

```diff
@@ -79,8 +79,10 @@ def kafka_server(kerberos):
 @pytest.fixture(scope="module", autouse=True)
 def kafka_client(kerberos):
     try:
-        kafka_client = client.KafkaClient("kafka-client")
-        kafka_client.install(kerberos)
+        kafka_client = client.KafkaClient(
+            "kafka-client", config.PACKAGE_NAME, config.SERVICE_NAME, kerberos
```

Comment:
Nit: Do we ever name this something other than `"kafka-client"`?

Reply:
So far as I could see yes but if we remove this option is not possible anymore to test multiple Kafka clusters even though it is currently not done.